settings.yml

```diff
@@ -68,7 +68,7 @@ variables:
   # ------------------------------- #
 
   bicepRegistryDoPublish: true # Set to true, if you would like to publish module templates to a bicep registry
-  bicepRegistryName: adpsxxazacrx001 # The name of the bicep registry (ACR) to publish to. If it does not exist, it will be created.
+  bicepRegistryName: skhanadpsxxazacrx001 # The name of the bicep registry (ACR) to publish to. If it does not exist, it will be created.
```

Comment:
Please undo the changes in this file as they are specific to your environment. 

In my experience the 'easiest' way to contribute is to open a PR from a branch so that you can rollback the changes in the settings.yml without needing to change it in your main branch.

Reply:
reverted changes in the settings.yml file as per suggestion